packet_queue/command.py

```diff
@@ -45,30 +64,67 @@ def configure(rest_server=False):
       help=('proxy port for receiving all inbound traffic if -luser'
             'is specified'))
   parser.add_argument(
-      '-p', '--port', type=int, required=True,
+      '-p', '--port', type=int, action='append', dest='ports', required=True,
       help='flaky inbound/outbound traffic occurs on specified port')
+  parser.add_argument(
+      '-b', '--bandwidth', type=int, default=-1,
+      help='The bandwidth in bytes per second')
+  parser.add_argument(
+      '-d', '--delay', type=int, default=0,
+      help='The one way delay in seconds')
+  parser.add_argument(
+      '-B', '--buffer', type=int, default=-1,
+      help='The size of the buffer in bytes')
+  parser.add_argument(
+      '-L', '--loss', type=verify_packet_drop_range, default=0.0,
+      help='The packet drop ratio. must be in range 0.0-1.0')
+  parser.add_argument(
+      '-D', '--direction', default='inbound',
+      choices=['inbound', 'outbound'],
```

Comment:
I meant that it could still be called "direction" but that the choices would be "client" and "server", even though that's a bit weird because client and server aren't really directions. "Orientation" might be better, actually?